src/library/scala/collection/immutable/Vector.scala

```diff
@@ -955,7 +954,7 @@ private[immutable] trait VectorPointer[T] {
 
     private[immutable] final def copyOf(a: Array[AnyRef]) = {
```

Comment:
I'm going for a bug fix with no visible changes (and binary compatibility), so while I agree that it's inconsistent, I don't think this is the time to change it.